flocker/node/agents/blockdevice.py

```diff
@@ -813,12 +830,12 @@ def destroy_volume(blockdevice_id):
         :return: A ``Deferred`` that fires when the volume has been destroyed.
         """
 
-    def attach_volume(blockdevice_id, host):
+    def attach_volume(blockdevice_id, storage_backend_id):
```

Comment:
Node agents only run stuff for current node. In future iterations we might want to have cloud agent that does attach for multiple nodes, but that is in The Future.